libs/platform/src/lib/components/form/multi-input/multi-input.component.spec.ts

```diff
@@ -0,0 +1,151 @@
+                name="reactiveFormInput"
+                zone="“zLeft”"
+                rank="1"
+                placeholder="Field placeholder text"
+            >
+                <fdp-multi-input
+                    [formControl]="ffl1.formControl"
+                    placeholder="Field placeholder text"
+                    type="text"
+                    id="fdp-id"
+                    name="input1"
+                    [listTemplate]="listTemplate"
+                    [listVariable]="selectedValue"
+                    (input)="searchData(searchString)"
+                    [(ngModel)]="searchString"
+                >
+                </fdp-multi-input>
+            </fdp-form-field>
+        </fdp-form-group>
+
+        <ng-template #listTemplate>
+            <fdp-list [noBorder]="true">
+                <fdp-standard-list-item
+                    *ngFor="let listItem of dropdownValues"
+                    [title]="listItem.name"
+                    (itemSelected)="itemSelected(listItem.name)"
+                >
+                </fdp-standard-list-item>
+            </fdp-list>
+        </ng-template>
+    `
+})
+class PlatformMulitiInputTest implements OnInit {
+    @ViewChild(PlatformMultiInputComponent)
+    multiInputComponent: PlatformMultiInputComponent;
+
+    _dataSource = new ListDataSource<Address>(new ListDataProvider());
+
+    selectedValue: string;
+
+    dropdownValues: Address[] | Observable<Address[]>;
+
+    public searchString: string;
+
+    @Input()
+    listTemplate: TemplateRef<any>;
+
+    constructor() {
+        this._dataSource.open().subscribe((data) => {
+            this.dropdownValues = data;
+        });
+    }
+    ngOnInit(): void {
+        this._dataSource.match('*');
+    }
+}
+
+describe('PlatformMultiInputComponent', () => {
+    let component: PlatformMulitiInputTest;
+    let fixture: ComponentFixture<PlatformMulitiInputTest>;
+    let multiInput: PlatformMultiInputComponent;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [PlatformMulitiInputTest, PlatformMultiInputComponent],
+            imports: [
+                CommonModule,
+                FormsModule,
+                PlatformMultiInputModule,
+                ReactiveFormsModule,
+                FdpFormGroupModule,
+                PlatformListModule,
+                StandardListItemModule,
+                RouterTestingModule
+            ],
+            providers: [DynamicComponentService, { provide: DATA_PROVIDERS, useClass: DataProvider as any }]
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(PlatformMulitiInputTest);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+        multiInput = component.multiInputComponent;
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should open and close the dropdown', () => {
```

Comment:
we should be using async tests. like` async ()=> {}`.
use await wait() to make test stable.